snuba/query/allocation_policies/errors_allocation_policy.py

```diff
@@ -0,0 +1,205 @@
+from __future__ import annotations
+
+import logging
+import time
+from typing import Any, cast
+
+from snuba import environment
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.query.allocation_policies import (
+    DEFAULT_PASSTHROUGH_POLICY,
+    AllocationPolicy,
+    QueryResultOrError,
+    QuotaAllowance,
+)
+from snuba.state import get_config
+from snuba.state.sliding_windows import (
+    GrantedQuota,
+    Quota,
+    RedisSlidingWindowRateLimiter,
+    RequestedQuota,
+)
+from snuba.utils.metrics.wrapper import MetricsWrapper
+
+metrics = MetricsWrapper(environment.metrics, "errors_allocation_policy")
+
+
+# A hardcoded list of referrers which do not have an organization_id associated with them
+# purposefully not in config because we don't want that to be easily changeable
+_ORG_LESS_REFERRERS = set(
```

Comment:
Why have this list? Why not skip any query that doesn't have an `organization_id` in it?

Reply:
some of them are valid, e.g. subscriptions. Some of them are maybe valid but I don't have enough knowledge. They don't constitute even a percent of traffic but I think you're right that we should make this list go away eventually